src/js/api/order.js

```diff
@@ -402,6 +402,105 @@ ripe.Ripe.prototype.setOrderStatus = function(number, status, options, callback)
     return this._cacheURL(options.url, options, callback);
 };
 
+/**
+ * Gets the order subscription status for the session user
+ *
+ * @param {Number} number The number of the order to get the subscription status
+ * @returns {Promise} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype.getSubscription = function(number, options, callback) {

```

Comment:
getOrderSubscription. Remember that this is accessible through `ripeSdk.getSubscription`, so for the consumer it's not clear you're talking about orders.